docs/DeveloperGuide.md

```diff
@@ -239,7 +238,7 @@ Sequence Diagram:
 
 #### Flow
 1. The user enters the command, eg. `edit eid/37 p/8461 4872 a/ntu`. The employee with Id 37 will be identified and will have its phone and address fields updated as specified, if it exists in SudoHR.
-2. The parser will instantiate a new `Phone` and `Address` object constructed from the arguments associated with `/p` and `/n`, which represent the new phone number and address fields respectively.
+2. The parser will instantiates a new `Phone` and `Address` object constructed from the arguments associated with `/p` and `/n`, which represents the new phone number and address fields respectively.
```

Comment:
thx for checking grammer but there no 's' for the verb that comes after the word 'will'